src/AutoRest.CSharp.V3/Pipeline/Extensions.cs

```diff
@@ -27,9 +27,9 @@ internal static class Extensions
             AllSchemaTypes.Dictionary => null,
             AllSchemaTypes.Duration => typeof(TimeSpan),
             AllSchemaTypes.Flag => null,
-            AllSchemaTypes.Integer => typeof(int),
+            AllSchemaTypes.Integer => null, // Handled in ToFrameworkType for NumberSchema
```

Comment:
remove them from here?

Reply:
This isn't a final design. I'm starting to get a handle of how we need to layer this stuff. Most of these mappings aren't going to hold up.